test/support/test_helpers.ex

```diff
@@ -1,15 +1,33 @@
 defmodule RemoteRetro.TestHelpers do
   use Wallaby.DSL
-  alias RemoteRetro.Repo
+  alias RemoteRetro.{Repo, User}
 
   def persist_idea_for_retro(context) do
-    %{idea: idea, retro: retro} = context
-    idea = Map.put(idea, :retro_id, retro.id)
+    %{idea: idea, retro: retro, user: user} = context
+    user =
+      if Map.has_key?(user, :id) do
+        user
+      else
+        Repo.get_by(User, email: user["email"])
+      end
+    idea =
+      Map.put(idea, :retro_id, retro.id)
+      |> Map.put(:user_id, user.id)
     idea = Repo.insert!(idea)
 
     Map.put(context, :idea, idea)
   end
 
+  def persist_user_for_retro(context) do
+    %{user: user} = context
```

Comment:
Not a blocker for this PR, but Elixir allows you to destructure a parameter, which would make sense here, as no other elements of `context` are used here